### PR TITLE
Fixed an edge-case scenario with ProjectItemModel::selectedBossImages()

### PR DESCRIPTION
**Pull request by twilson271828**
There was a problem with the selection criteria in ProjectItemModel.  When a user clicks on an image list, but also selects some but not all of the images in the image list, selectedBossImages() was returning all of the images in the image list.  This PR fixes that problem.

### CODE REVIEW

**Comment by twilson271828** [comment]
Git added my most recent PR (updateBundleObservationSolveSettings function) to this one .  That wasn't my intention, but I guess it won't harm anything.

File: isis/src/qisis/objs/JigsawSetupDialog/JigsawSetupDialog.cpp [hunk 1]
```diff
@@ -782,6 +785,144 @@ namespace Isis {
     return QString(m_ui->inputControlNetCombo->currentText());
   }
```
Comment (ihumphrey) [inline isis/src/qisis/objs/JigsawSetupDialog/JigsawSetupDialog.cpp:787]: Make sure to document the updateBundleObservationSolveSettings method here

File: isis/src/qisis/objs/JigsawSetupDialog/JigsawSetupDialog.h [hunk 2]
```diff
@@ -156,10 +157,11 @@ namespace Isis {
...
 
     private:
-    void makeReadOnly()    ;
+    void makeReadOnly();
     void fillFromSettings(const BundleSettingsQsp settings);
     void showTargetParametersGroupBox();
     void hideTargetParametersGroupBox();
+    void updateBundleObservationSolveSettings(BundleObservationSolveSettings &boss);
 
     void createObservationSolveSettingsTreeView();
```
Comment (ihumphrey) [inline isis/src/qisis/objs/JigsawSetupDialog/JigsawSetupDialog.h:171]: I would add a history comment about adding updateBundleObservationSettings and what it does

**Review by ihumphrey — COMMENTED** [review]
Looks good! Just be sure to add history to JigsawSetupDialog.h and Doxygen documentation to JigsawSetupDialog::updateBundleObservationSolveSettings

File: isis/src/qisis/objs/JigsawSetupDialog/JigsawSetupDialog.cpp (hunk 1 above)
Comment (twilson271828) [reply isis/src/qisis/objs/JigsawSetupDialog/JigsawSetupDialog.cpp:787]: Added a history entry + function documentation for the method.

**Comment by ihumphrey** [comment]
:+1:

**Pull request merged**